Makefile

```diff
@@ -6,20 +6,14 @@
 install-hooks: .git/hooks/pre-commit
 	@true
 
-regenerate-samples:
```

Comment:
Please don't remove this target as it's handy when regenerating all the samples.

Reply:
With the composite build, you should be able to just run `./gradlew generateSwagger` and it should regenerate the samples.
That's actually a more general question I had: why not making everything in gradle instead of `Makefile`? It feels a bit awkward to have two build systems.

Reply:
@macisamuele 👆

Reply:
That's a concern I already raised with @macisamuele. I'm also in for having composite build tasks rather than Gradle + Make. I generally don't have a really strong opinion on this and I'm fine with whatever makes everyone comfortable :)

Reply:
Let's merge this but keep the Make+Gradle discussion opened. I would love to get @macisamuele 's opinion on this.

Reply:
Replacing `make regenerate-samples` with `./gradlew generateSwagger` looks like a very good to me.
I'll just, in a future PR, remove the  `regenerate-samples` `PHONY` (first Makefile line) 😅